pom.xml

```diff
@@ -49,15 +51,20 @@
             <artifactId>jakarta.validation-api</artifactId>
             <version>3.0.2</version>
         </dependency>
+        <dependency>
+            <groupId>jakarta.persistence</groupId>
+            <artifactId>jakarta.persistence-api</artifactId>
+            <version>3.1.0</version>
+        </dependency>
```

Comment:
I don't see why this dependency started to be necessary

Reply:
I am not sure. I must have added it to the wrong repo by mistake. Removed now